.github/workflows/build_and_deploy_infer.yml

```
@@ -22,6 +22,12 @@ jobs:
   validate:
     runs-on: ubuntu-latest
 
+    # Checkoutしていないため、そのままだとworking-directoryが存在しないディレクトリを指して落ちる。
+    # そのため、job単位でリセットする。
+    defaults:
+      run:
+        working-directory: .
```

（細かいけど）

内容のコメントなのかメモ書きなのか分かりやすいように、メモ書きだということをメモしておくとより読みやすいかも！

```suggestion
    # NOTE:
    # Checkoutしていないため、そのままだとworking-directoryが存在しないディレクトリを指して落ちる。
    # そのため、job単位でリセットする。
    defaults:
      run:
        working-directory: .
```